### ISSUE TITLE
JSJoda is not defined

### ISSUE THREAD

**Issue by jgrodziski** [juxt/tick#45]
In latest tick release (0.4.16-alpha), running tests on ChromiumHeadless throws an error
```
HeadlessChrome 76.0.3803 (Mac OS X 10.14.5) ERROR
  Uncaught ReferenceError: JSJoda is not defined
  at target/test.js:69318:20

  ReferenceError: JSJoda is not defined
      at target/test.js:69318:20
```
The tests runs perfectly with 0.4.14-alpha, we use shadow-cljs for the build of the JS bundle and js-joda is correctly defined with npm with version "1.11.0", what's different in latest tick release that could induce that error?

**Comment by henryw374** [comment]
I think I know what the problem is. the js-joda foreign lib now has global exports, and cljs.java-time now requires a namespace 'js-joda', instead of cljsjs.js-joda. So the cljsjs.js-joda shim ns I presume you're getting from shadow-cljsjs is not being used. Can you try creating this shim ns in your own project and see if that works?

```
(ns js-joda
  (:require ["js-joda" :as jsjoda]))

(js/goog.exportSymbol "JSJoda" jsjoda)
```

**Comment by henryw374** [comment]
ok, this require change is definitely the problem. 

I'll fix up cljs.java-time so it no longer refers to the global JSJoda, and do global exports for the other addon libs, and then shadow users can drop the extra shadow-cljsjs dependency

**Comment by jgrodziski** [comment]
The exportSymbol doesn't work because I think it is evaluated after the JSJoda stuff.
I agree with the solution of using the cljsjs dependency, shadow-cljs users can then drop in the `shadow.cljsjs` dependency to be able to require:`(require [cljsjs.js-joda :as js-joda]`.

**Comment by henryw374** [comment]
this should be fixed in 0.4.17-alpha. please reopen if you have any problems.
